goss-testing/automated/run-ncn-tests.sh

```diff
@@ -44,6 +44,20 @@ function is_pit_node {
     [[ -f /etc/pit-release ]]
     return $?
 }
+ 
+function is_vshasta_node {
+    # This is the best check for an image specifically booted to vshasta
+    if [[ -f /etc/google_system ]]; then
+      return $?
```

Comment:
Should be the same, but seems more straightforward to not use the variable when we always know it is 0?

```suggestion
      return 0
```